Update to 0.6.4

jan-janssen opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jan-janssen committed:
Update to 0.6.4

jan-janssen commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `tests[].python.python_version` or `tests[].requirements.run` section of the recipe, you should usually use the pin `python_version: ${{ python_min }}.*` or `python ${{ python_min }}.*` for the `python_version` or `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16435455370](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16435455370). Examine the logs at this URL for more detail.</sub>

Commit:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.21.12.24.54

jan-janssen committed:
Update recipe.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

jan-janssen committed:
Update recipe.yaml

jan-janssen committed:
Update recipe.yaml

jan-janssen committed:
Update recipe.yaml

jan-janssen committed:
Update recipe.yaml

jan-janssen committed:
Update recipe.yaml

jan-janssen commented:
@raybellwaves The pull request is now ready for your review. I updated the package to version 0.6.4 and added myself as maintainer.

jan-janssen commented:
@conda-forge/core The recipe seems to be inactive from the beginning of the year with no pull requests getting merged. So I fixed the latest version and added myself as maintainer. Can one of you merge this pull request?

hmaarrfk commented on recipe/recipe.yaml:
```diff
@@ -18,39 +18,29 @@ build:
-    - pydantic >=1.10,<3,!=2.6.0
-    - docker-py
-    - packaging
+    - autogen-core =0.6.4
```
```suggestion
    - autogen-core =${{ version }}
```

Should this just be dynamically updated to match the version?

jan-janssen replied:
Good catch - thanks for the suggestion.

jan-janssen committed:
Update recipe/recipe.yaml

Co-authored-by: Mark Harfouche <mark.harfouche@gmail.com>
